server/trace-func-go/trace_func.go

```diff
@@ -62,5 +62,5 @@ func main() {
 		log.Infof("Function type: EMPTY\n")
 	}
 
-	standard.StartGRPCServer("", serverPort, functionType, *zipkin)
+	standard.StartGRPCServer("", serverPort, functionType, *zipkin, false)
```

Comment:
Can you extract the vSwarm from here? I don't like that we are introducing this vSwarm greeter into our Invitro workload. Since we are creating it just for testing, we don't need to reuse all of the features of the workload that we have (tracing, handling SIGTERM); you can just put this mock server into test file instead. Or if you can, you can try to reuse the relay server from vSwarm, not to duplicate the code.

Reply:
Fixed this, fixed the formatting as well and rebased on main. Please review again.

Thanks

Reply:
I don't think you did what I was asking for.

Since we use `StartVSwarmGRPCServer` only in tests, we don't need anything from it except the ability to accept and respond with VSwarm protobufs. So, you can significantly slim it down by removing everything unnecessary for the simplest gRPC server (signal handling, `TraceFunctionExecution`, tracing).